.gitlab.mk

```diff
@@ -110,20 +109,29 @@ deploy_prepare:
 	rm -rf build
 
 package: deploy_prepare
+	# Set PRODUCT_NAME for the Package itself.
+	if [ "$$(echo $(GC64) | sed 's/.*=//')" = ON ]; then                                                           \
+		export PRODUCT_NAME=tarantool-gc64; \
+		$(shell sed -i'' -e 's/Package: tarantool$$/Package: tarantool-gc64/' debian/control) \
```

Comment:
I think we need to add `if` statement here and execute it if OS is `Debian` or `Ubuntu`.

Reply:
done